.circleci/config.yml

```diff
@@ -26,12 +26,15 @@ jobs:
             - cargo-index-v2-{{ checksum ".circleci/crates.io-index.head" }}
             - cargo-index-v2-
       - run:
-          name: Generate Cargo.lock and update cargo registry index
+          name: Generate Cargo.lock
           command: |
             cargo generate-lockfile
-            git -C /home/circleci/.cargo/registry/index/github.com-1ecc6299db9ec823 \
-                show-ref -s refs/remotes/origin/master |
-              tee .circleci/crates.io-index.head
+      - run:
+          name: Update head hash of cargo registry index
+          command: |
+            cat /home/circleci/.cargo/registry/index/*/.git/FETCH_HEAD |
```

Comment:
Shall we use the same approach in our GH Actions pipeline? No unnecessary accesses to GitHub servers which is nice.

Reply:
GitHub Actions does not use this, as their cache [can't be saved with an updated key](https://github.com/actions/cache/issues/135). The HTTP request for the head hash still works.